src/renderer/fonts/caching_shaper.rs

```diff
@@ -42,8 +42,16 @@ impl CachingShaper {
         let font_key = self
             .options
             .as_ref()
-            .map(|options| options.fallback_list.first().unwrap().clone().into())
-            .unwrap_or(FontKey::Default);
+            .map(|options| FontKey {
+                italic: options.italic,
+                bold: options.bold,
+                font_selection: options.fallback_list.first().unwrap().clone().into(),
+            })
+            .unwrap_or(FontKey {
```

Comment:
Maybe `.unwrap_or(FontKey::Default);`

Reply:
This is kind of what my [TODO above](https://github.com/Kethku/neovide/pull/769#discussion_r659818432) is about . . . it would in theory be cleaner to impl `Default` for `FontKey`, but that can only be used here; below it would require adding a method like `.with_font_selection(selection)` which I'm not sure how I feel about.

I don't really have any strong opinions on which way is better, I'm happy to add a method or two on `FontKey`, I just feel like this way is simpler.

Reply:
I think default option is used at least in 2 places so yeah, I'd recommend to implement it to avoid duplicate logic, even if it's really small. :p
But maybe @Kethku can say more which he prefers...